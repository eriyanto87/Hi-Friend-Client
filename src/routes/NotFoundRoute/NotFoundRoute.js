import React from "react";
import "./NotFoundRoute.css";
import Footer from "../../components/Footer/Footer";

export default function NotFoundRoute() {
  return (
    <section className="notfound">
      <h1>Hi, Friend</h1>
      <h2>404 - Page Not Found</h2>
      <p>Try going back to the previous page</p>
      <Footer />
    </section>
  );
}
