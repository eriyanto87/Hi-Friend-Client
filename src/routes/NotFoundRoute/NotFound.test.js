import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import NotFoundRoute from "./NotFoundRoute";

describe("NotFound component", () => {
  it("NotFound Snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <NotFoundRoute />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <NotFoundRoute />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
