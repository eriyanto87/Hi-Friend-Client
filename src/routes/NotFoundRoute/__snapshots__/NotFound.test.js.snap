// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component NotFound Snapshot 1`] = `
<section
  className="notfound"
>
  <h1>
    Hi, Friend
  </h1>
  <h2>
    404 - Page Not Found
  </h2>
  <p>
    Try going back to the previous page
  </p>
  <div
    className="footer"
  >
    Â© 2020
  </div>
</section>
`;
